GenumKit/Parsers/StringsFileParser.swift

```diff
@@ -17,12 +31,41 @@ public final class StringsFileParser {
 
   // Localizable.strings files are generally UTF16, not UTF8!
   public func parseStringsFile(path: String) throws {
+    guard let data = NSData(contentsOfFile: path) else {
+      throw StringsFileParserError.FailureOnLoading(path: path)
+    }
+
+    let plist = try NSPropertyListSerialization.propertyListWithData(data, options: NSPropertyListReadOptions.Immutable, format: nil)
+
+    guard let dict = plist as? Dictionary<String, String> else {
+      throw StringsFileParserError.InvalidFormat
+    }
+
     var encoding: NSStringEncoding = NSUTF16StringEncoding
     let fileContent = try NSString(contentsOfFile: path, usedEncoding: &encoding)
+
+    func extractKey(line: String) -> String? {
```

Comment:
The more I think about it the less I'm sure people care about the order, as:

* they probably never go read the generated file, but rather rely on auto completion
^ most people often use reverse-DNS naming for their keys, like `"home.header.title"` so sorting alphabetically would still keep related keys together

At my company we actually _do_ sort our `Localizable.strings` file alphabetically (they are auto-generated by a tool of ours exporting localization from the POEditor service), so…